CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.4.0
```

Comment:
Too good that there is a CHANGELOG.md in this repo? I didn't notice it

Reply:
It's annoying since it duplicates the informations
but it's mandatory for pub.dev